assets/src/edit-story/elements/shape/display.js

```diff
@@ -22,19 +22,28 @@ import styled from 'styled-components';
 /**
  * Internal dependencies
  */
-import { elementFillContent, elementWithBackgroundColor } from '../shared';
+import {
+  elementFillContent,
+  elementWithBackgroundColor,
+  elementWithBorderRadius,
+} from '../shared';
 import StoryPropTypes from '../../types';
 
 const Element = styled.div`
   ${elementFillContent}
   ${elementWithBackgroundColor}
+  ${elementWithBorderRadius}
 `;
 
-function ShapeDisplay({ element: { isDefaultBackground, backgroundColor } }) {
+function ShapeDisplay({
+  element: { isDefaultBackground, backgroundColor, borderRadius },
+}) {
   if (isDefaultBackground) {
     return <Element />;
   }
-  return <Element backgroundColor={backgroundColor} />;
+  return (
+    <Element backgroundColor={backgroundColor} borderRadius={borderRadius} />
```

Comment:
Here you don't check if the mask supports it - which I also don't think is necessary. But we should be consistent.